prow/github/client.go

```diff
@@ -594,14 +610,14 @@ func (c *client) requestRetry(method, path, accept string, body interface{}) (*h
 	var hostIndex int
 	var resp *http.Response
 	var err error
-	backoff := initialDelay
-	for retries := 0; retries < maxRetries; retries++ {
+	backoff := c.initialDelay
+	for retries := 0; retries < c.maxRetries; retries++ {
 		if retries > 0 && resp != nil {
 			resp.Body.Close()
 		}
 		resp, err = c.doRequest(method, c.bases[hostIndex]+path, accept, body)
```

Comment:
What if we have a version of the client call you're trying to make like `GetFileMayNotExist()` which explicitly allows 404 as a response?